hub-search-proxy/test/mock_query_basic.json

```diff
@@ -0,0 +1,9 @@
+{
+    "event": {
+        "httpMethod": "POST",
+        "headers": { "Content-Type": "application/json" },
+        "body": {
+            "comment": "Tis better to have tested and lost than never to have tested at all"
```

Comment:
Tis true

Reply:
Thanks @warrickct , I thought I was pretty philosophical here #im14andthisisdeep

Reply:
@Hganavak thank you Sam, gave me a chuckle when I first ran the test